ci/azure/posix.yml

```diff
@@ -90,3 +90,8 @@ jobs:
           Write-Error "$($matches[1]) tests failed"
         }
       displayName: Check for test failures
```

Comment:
Can you have this in quotes as the rest of `displayName` properties in the fie? Looks like they are not needed, but better to have it consistent.